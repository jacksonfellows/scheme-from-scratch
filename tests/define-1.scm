(define (mymap f lst)
  (if (null? lst)
      '()
      (cons (f (car lst)) (mymap f (cdr lst)))))

(define (times100 x)
  (fx* x 100))

(mymap times100 '(-3 -2 -1 0 1 2 3))
