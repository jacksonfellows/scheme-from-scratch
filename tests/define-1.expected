(-300 -200 -100 0 100 200 300)
